Dockerfile

```diff
@@ -0,0 +1,6 @@
+# syntax=docker/dockerfile:1
+
+FROM scratch
```

Comment:
Maybe we should use `alpine` instead? That gives ability for users to install additional commands with `apk` if needed.

Reply:
Also, I think this scratch image will not be able to install any fabric plugin from the registry because it's missing CA certs:
```
/etc/ssl/certs/ca-certificates.crt
```